pkg/git/repo.go

```diff
@@ -35,7 +35,8 @@ var (
 type Repo interface {
 	GetPath() string
 	GetClonedBranch() string
-	Copy(dest string) (Repo, error)
+	Copy(dest string) (Worktree, error)
+	CopyToModify(dest string) (Repo, error)
```

Comment:
[IMO] How about using the `Worktree` and `Copy`?

```
	Worktree(dest string) (Worktree, error)
	Copy(dest string) (Repo, error)
```

Reply:
It sounds clear, but I want to leave here as-is.
I want to clear when to use the `CopyToModify` method with its naming.
Almost all of implementation, we don't need modification of git commits/histories, so we should not use CopyToModify, and we can know from its naming.
If we rename them as Worktree and Copy, some implementation may call Copy method mistakenly even if it's enough to call the Worktree method.

WDYT?

Reply:
> Almost all of implementation, we don't need modification of git commits/histories, so we should not use CopyToModify, and we can know from its naming.

I got your point! I agree with it.